pom.xml

```diff
@@ -208,6 +211,7 @@
     <module>junit/kubernetes-junit-jupiter</module>
     <module>junit/kubernetes-junit-jupiter-autodetected</module>
     <module>junit/kubernetes-server-mock</module>
+    <module>junit/kube-api-test</module>
```

Comment:
should be added together with the Java11 modules, otherwise Java8 compilation won't work

Reply:
moved. thx